### PR TITLE
Add Travis and other automatic testing

### PR DESCRIPTION
**Pull request by tuchfarber**

### CODE REVIEW

**Comment by staubina** [comment]
@kaizoku @0x29a

File: tox.ini [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [tox]
-envlist = py35, py38, flake8
```
Comment (0x29a) [inline tox.ini:2]: py3.5 added here intentionally after [this comment](https://github.com/edx/cc2olx/pull/1#discussion_r455983172) in https://github.com/edx/cc2olx/pull/2/commits/5ed1eaa00ebdb0bf5369ac9db88b036f377abbd7 as well as some type casts of path for functions that lack support of `pathlib`.

And indeed, `lms` container of the devstack has python with version `3.5.2`.

**Comment by kaizoku** [comment]
The tests runners look good to me. I'd like to test with the updated requirements, and the python requirement in my devstack, though my devstack is reprovisioning at the moment. I'll have some time to test installation and conversion tomorrow.

**Comment by tuchfarber** [comment]
I've move the files back to 3.5/3.8 instead of 3.6. Re-ran `make upgrade` and requirements files didn't change, so I'm assuming they're all 3.5/3.6 compatible. I'm out tomorrow, but @staubina can merge this once you're good with it :+1:

**Comment by 0x29a** [comment]
Thanks, @tuchfarber.

**Comment by kaizoku** [comment]
Thanks for updating that @tuchfarber.
I tested installing the test requirements in my devstack. We won't need to ordinarily install the dev and test requirements in the devstack, but if we add other runtime requirements later we'll want to be sure they're 3.5 compatible for integration.

twine 3.2.0 wasn't compatible with python <3.6, and after running `make upgrade` from python 3.5.9 I saw some changes to requirements that made them compatible I'm not against merging this as is, since it works fine for the test runners, but should I add the commit with the requirements changes I'm getting?

**Comment by staubina** [comment]
@kaizoku, @tuchfarber is unavailable today. I think he is looking at putting 3.5 back into the files and doing a little local testing. We will update Monday.

**Comment by tuchfarber** [comment]
I tried running `make upgrade` using 3.5.9 from deadsnakes, but the lack of requirements changing makes me think it didn't work correctly. I'd be fine if someone commits the requirement upgrades on this branch since I'm out. If not I'll tackle it on Monday

**Comment by 0x29a** [comment]
@tuchfarber
>I'd be fine if someone commits the requirement upgrades on this branch since I'm out. If not I'll tackle it on Monday

Have no rights to push in this branch.

You can cherry-pick this: https://github.com/open-craft/cc2olx/commit/b25dbb9910ff6b5aea55a9b8d7d496165c2511d4

**Comment by tuchfarber** [comment]
Sorry about that. I got the requirements back to 3.5 compatible (note twine back to `1.15`). Waiting for tests to run, then I'll squash and merge.

**Comment by kaizoku** [comment]
No worries @tuchfarber. The twine requirement was the primary one not compatible with 3.5.
Thanks again for configuring the test runners!

**Pull request merged**